p2p/src/node.rs

```diff
@@ -505,6 +527,32 @@ where
                 }
                 Ok(None)
             }
+            SwarmEvent::ExternalAddrConfirmed { address } => {
+                if let Err(err) = self
+                    .peers_tx
+                    .send(peers::Message::NewLocalAddress(address))
```

Comment:
there's not really any recovering here right? if the task shutdown/channel closes.. nothing is going to start it again so we'll just keep logging these indefinitely. is there some shutdown we should do or is this fine for now?

Reply:
Should be fine for now. The task that process these is long lived and only stops when the whole peer is being stopped.